### PR TITLE
Feature/backend lang alias

### PR DESCRIPTION
**Pull request by a-bougouin**
### Purpose
Allow user to use custom lang code.

### Comment
User must add setting, for instance:
```
custom_lang_codes = {
  'ja' => 'jp'
}
```

### CODE REVIEW

File: lib/wovnrb/lang.rb [hunk 1]
```diff
@@ -78,6 +81,7 @@ def lang_code
     # @return [String]                 URL added langauge code.
     def add_lang_code(href, pattern, headers)
       return href if href =~ /^(#.*)?$/
+      lang_code = Store.instance.settings['custom_lang_aliases'][@lang_code] || @lang_code
```
Comment (a-bougouin) [inline lib/wovnrb/lang.rb]: Which name do you want to change?

Comment (mrasu) [inline lib/wovnrb/lang.rb]: lang_code

Comment (a-bougouin) [inline lib/wovnrb/lang.rb]: I changed, if you have better idea please suggest

**Comment by mrasu** [comment]
please accept symbol

**Comment by a-bougouin** [comment]
@mrasu I fixed setting hash

**Pull request merged**